### ISSUE TITLE
Only pulling 1st page of library / 20 titles

### ISSUE THREAD

**Issue by Narwalrus** [joonaspaakko/Audible-Library-Extractor-Documentation#1]
Only the most recent 20 titles are being pulled from the library. 
I also tried adding all the titles to a collection and having that export and it also only pulled the most recent 20 books.

Tried on both Firefox / Edge without any other extensions enabled

**Comment by joonaspaakko** [comment]
Does the extraction "freeze" at 20 books or does it seem to successfully extract 20 books and open the gallery?

**Comment by joonaspaakko** [comment]
@Narwalrus

**Comment by Narwalrus** [comment]
It seems to extract max 20 books successfully, doesn't freeze.
I tried adding titles in further pages to a collection, with some from the
first page and it only pulled the titles from the first page.

**Comment by joonaspaakko** [comment]
@Narwalrus, alright. The only thing I can say for sure is that I did a few tests with the extension v.0.2.8 (latest release) and I didn't experience the same issue. Sometimes something changes in Audible and everyone will be hitting the same issue, but this time it seems like it's something specific in your library. This kind of stuff has happened before when the extension assumes it received data or certain type of data but with certain libraries with specific set of data it may cause the extension to fail. Though skipping pages like this is a bit unusual.

**If you could do the following in Chrome:**

0. Download this build of the extension [:arrow_down: audible-library-extractor-v0.0.0.zip](https://github.com/joonaspaakko/Audible-Library-Extractor-Documentation/files/8826542/audible-library-extractor-v0.0.0.zip)
1. Go to the address:  `chrome:extensions`.
	- Or:  `Window > Extensions`
2. Disable any previous installations of ALE.
3. Turn on the developer mode from the top right (If needed)
4. Drag the downloaded `.zip` file in the browser window to install
5. Try running the extraction.
    - I've hidden the settings, just click the extract button.
6. When the extraction is done and you see the `Opening the gallery after packing up the data...` message, it won't be opening the gallery...
7. Open Chrome dev tools console (Shortcuts below) 
    - <table>
        <thead>
          <tr>
            <th>Action</th>
            <th>Mac</th>
            <th>Windows / Linux</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>Open the <b>Console</b> panel</td>
            <td><kbd>Command</kbd>+<kbd>Option</kbd>+<kbd>J</kbd></td>
            <td><kbd>Control</kbd>+<kbd>Shift</kbd>+<kbd>J</kbd></td>
          </tr>
        </tbody>
      </table>
8. Save the log as a file.
    - Basically just right-click anywhere in the console with the messages and in the context menu pick "Save as..."
    - More details if needed: https://support.google.com/docs/thread/1873663/collecting-console-logs-chrome-browser-only?hl=en
9. Send the log file to my email `paakkoj@gmail.com`.

And hopefully the log file will tell me something...
